Sources/PointFree/Episode/Episode.swift

```diff
@@ -122,11 +122,20 @@ public struct Episode {
     public enum BlockType: Equatable {
       case code(lang: CodeLang)
       case correction
-      case image(src: String)
+      case image(src: String, sizing: ImageSizing)
       case paragraph
       case title
       case video(poster: String, sources: [String])
 
+      public static func image(src: String) -> BlockType {
```

Comment:
this provides backwards compatibility with all the `.image`'s out in the wild right now.